client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/useDesksFilterCard.ts

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+import logger from 'logger/logger';
+import Desk from 'models/Desk';
+import { Severity } from 'models/Logger';
+import { useEffect, useState } from 'react';
+
+const useDesksFilterCard = () => {
+    
+    const [desks, setDesks] = useState<Desk[]>([]);
+    const [isLoading, setIsLoading] = useState<boolean>(true);
+
+    const fetchDesks = () => {
```

Comment:
This function is called once yet at each re-render we will re-declare it. 
To fix this you can move all the fetch logic outside of the hook and keep the hook logic (setDesks, setIsLoading) in the useEffect.